src/components/dateTimePicker/index.js

```diff
@@ -93,7 +93,11 @@ class DateTimePicker extends Component {
     /**
      * style to apply to the iOS dialog header
      */
-    headerStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number])
+    headerStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+    /**
+     * Render custom expandable input (requires expandable to be true)
+     */
+    renderExpandableInput: PropTypes.elementType
```

Comment:
while for the TextField it made sense to call the prop `renderExpandleInput` I think in this case it's less relevant to the user what's expandable is. 
I think renderInput is enough